charts/tornjak/templates/server-statefulset.tpl

```diff
@@ -1,3 +1,22 @@
+{{- if .Values.tornjak }}
+{{- if .Values.tornjak.config }}
+{{- if .Values.tornjak.config.enableUserMgment }}
```

Comment:
maybe would be good to rename to enableFrontend, which should be separate from enableUserMgment (this is toggled when the authUri is set